src/app/app.component.ts

```diff
@@ -2,9 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'bb-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  template: '<router-outlet></router-outlet>',
```

Comment:
sometimes it is a bad solution, just write <router-outlet></router-outlet> in app template and that's all